src/test/java/org/folio/rest/impl/UsersAPIIT.java

```diff
@@ -145,4 +165,23 @@ void facetsLimit0() {
   void facetsLimit1() {
     facets(1);
   }
+
+  @Test
+  void delete() {
+    String id1 = "114b14e8-422d-429a-b89e-b55c439b3b21";
+    String id2 = "22531038-08b8-4c71-a72c-9d48f63f3682";
+    String id3 = "13836e0e-4331-4386-a8be-2345eece6de3";
```

Comment:
Personally, I wouldn't hardcode these and would use a randon UUID each time.

Reply:
username == "1*" only works as intended if id1 and id3 start with 1 and id2 don't.

Using a random UUID doesn't result in consistent tests, sometimes it fails, sometimes it succeeds.

Reply:
I suggest defining the details that the tests relies on explicitly, in this case the username instead of inferring it from something the record ID that is incidental and should be different each time (as this is what we would expect in real use).